pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/UnusedPrivateField.xml

```diff
@@ -531,7 +531,7 @@ public class Foo {
     </test-code>
     <test-code>
         <description>#1420 UnusedPrivateField: Ignore fields if using lombok - 7</description>
-        <expected-problems>1</expected-problems>
+        <expected-problems>0</expected-problems>
```

Comment:
I'm not sure if  should throw a violation in this case.

Reply:
It's ok, it shouldn't flag this as a violation